docs/serviceregistry/manuals/temperature-http-demo.md

```diff
@@ -0,0 +1,846 @@
+- **interfaces:** All the services use http protocol, so we will go with the template named generic-http, that already exists in the Local Cloud. Note that in our case, the service discovery interface policy is set to _restricted_, which means that only already existing interface templates can be used. If you set this to _extendable_ or _open_, you can use non-existent interface templates, and they will be created as well. The interface provided by the Kelvin info service is the following:
+  - GET tp2.greenhouse.com:8080/kelvin/query
+  
+Based on these specifications, the request looks like this:
+
+~~~
+curl -X 'POST' \
+  'http://localhost:8443/serviceregistry/service-discovery/register' \
+  -H 'accept: application/json' \
+  -H 'Authorization: Bearer SYSTEM//temperature-provider2' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "serviceDefinitionName": "kelvin-info",
+  "version": "",
+  "expiresAt": "2030-01-01T00:00:00Z",
+  "metadata": {
+    "margin-of-error": 0.5
+  },
+  "interfaces": [
+    {
+      "templateName": "generic-http",
+      "protocol": "http",
+      "policy": "NONE",
+      "properties": {
+        "accessAddresses": ["192.168.56.116", "tp2.greenhouse.com"],
+        "accessPort": 8080,
+        "basePath": "/kelvin",
+        "operations": {"query-temperature": { "method": "get", "path": "/query"} }
+      }
+    }
+  ]
+}'
+~~~
+
+**2. Celsius info:** 
+The only difference with the Kelvin info is the service definition name (celsius-info) and the interface (GET tp2.greenhouse.com:8080/celsius/query). All the other registation data will remain the same.
+  
+So the request will look like this:
+
+~~~
+curl -X 'POST' \
+  'http://localhost:8443/serviceregistry/service-discovery/register' \
+  -H 'accept: application/json' \
+  -H 'Authorization: Bearer SYSTEM//temperature-provider2' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "serviceDefinitionName": "celsius-info",
+  "version": "",
+  "expiresAt": "2030-01-01T00:00:00Z",
+  "metadata": {
+    "margin-of-error": 0.5
+  },
+  "interfaces": [
+    {
+      "templateName": "generic-http",
+      "protocol": "http",
+      "policy": "NONE",
+      "properties": {
+        "accessAddresses": ["192.168.56.116", "tp2.greenhouse.com"],
+        "accessPort": 8080,
+        "basePath": "/celsius",
+        "operations": {"query-temperature": { "method": "get", "path": "/query"} }
+      }
+    }
+  ]
+}'
+~~~
+
+**2. Alert service:** 
+Our last service will be responsible for sending error messages. The registration data is the following:
+- **service definition name:** In this case this is alert-service.
+- **version:** We will use the default version.
+- **expires at:** The alert service expires a bit earlier than the previous ones, so we set this to 01. 01. 2025.
+- **metadata:** For alert service, the maximum possible delay is given, which is 15 sec.
+- **interfaces:** The interfaces provided by the alert service are the following:
```

Comment:
There is no multiple interfaces (as we understand interface in Arrowhead). So I'd rephrase this sentence: The interface's endpoints provided by ...